assets/src/edit-story/components/panels/videoAccessibility.js

```diff
@@ -28,12 +27,16 @@ import { __ } from '@wordpress/i18n';
 /**
  * Internal dependencies
  */
-import { Media, Row, usePresubmitHandler } from '../form';
-import { Note, ExpandedTextInput } from './shared';
+import { Row, usePresubmitHandler } from '../form';
 import { SimplePanel } from './panel';
 import { getCommonValue, useCommonObjectValue } from './utils';
+import { ExpandedTextInput, Note } from './shared';
+
+const DEFAULT_RESOURCE = {
+  poster: null,
```

Comment:
Still necessary now that poster is removed from this panel?

Reply:
Updated.